### PR TITLE
Create create-redirects.sh

### PR DESCRIPTION
**Pull request by austinkregel**

### CODE REVIEW

**Comment by daspecster** [comment]
What is this for exactly?

File: create-redirects.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
+
+# Author: Austin Kregel <github@kbco.me>
```
Comment (daspecster) [inline create-redirects.sh]: Can you put some kind of description about what this script actually does?

Comment (austinkregel) [reply create-redirects.sh]: Sure can :) 

The other one didn't, so I didn't know if there was a pattern you wanted me to follow.

**Review by daspecster — APPROVED** [review]

**Comment by austinkregel** [comment]
@daspecster you have to merge this if you accepted it 😄

**Pull request merged**